code/datums/ai/hauntium/hauntium_subtrees.dm

```diff
@@ -2,6 +2,8 @@
 	var/obj/item/item_pawn = controller.pawn
 
 	if(ismob(item_pawn.loc)) //We're being held, maybe escape?
+		if(controller.blackboard[BB_LIKES_EQUIPPER])//don't unequip from people it's okay with
+			return SUBTREE_RETURN_FINISH_PLANNING
```

Comment:
dont tell it to finish planning if this plan is not actually doing anything. If you finish planning it should finish with a reason.

Reply:
Shouldn't it be inert?